### DISCUSSION TITLE
point filter now has a deadzone slider

### DISCUSSION THREAD

**Discussion by sthalik** [opentrack/opentrack#1455] [General]
The point filter included in the PT tracker can help Accela with the kinds of noise it's not good at. It's based on a simplified Accela curve and now has a deadzone slider as well. The idea is that small distortions in point positions (that the point filter takes care of) can cause large jumps in the estimated pose (that Accela has to contend with). Therefore filtering points themselves requires less filtering than pose filtering.

Consider trying out the point filter instead of Accela as the main filter. The PF sensitivity should be between 0.1 and 0.2 (0.15 is a good start), and the deadzone should be below 0.10 (which is a good start). You might want to keep using Accela purely to increase smoothness (deadzone of 0.01 or zero, and sensitivity at 0.5 or below) but PF should take care of most of the instability.

Test build with PF deadzone: <<https://www.dropbox.com/s/otye58zu09ukzkl/opentrack-master-20220419_01.7z>>

Please share your results and experiences.
-sh

**Comment by dglava** [comment]
I have decided to try this out and I didn't get acceptable results with the point filter alone. I didn't test it extensively and I'm not even sure I did everything right. I tried the recommended settings mentioned above (I'm not sure which one of the sliders the sensitivity one is). It was jittery while holding still. Then I tried to adjust the sliders until I got a somewhat acceptable result, but the only way to get rid of the "holding still jitter" was to go to extremes and make it rather "slow".

I am using a PS3Eye camera with the free drivers.

With the recommended settings and jitter:
Settings: https://ibb.co/rwFdN3s
Video: https://streamable.com/apzxuw

Adjusted settings, somewhat jitter free, but slow and unresponsive:
Settings: https://ibb.co/KDxQ6ZZ
Video: https://streamable.com/5eilaw

**Reply by darmach** [reply]
@sthalik how about the Limit parameter? What does it do?
<img width="622" height="131" alt="image" src="https://github.com/user-attachments/assets/33071ad5-3285-4e8b-af6c-ad250fd0990f" />

**Reply by sthalik** [reply]
It clamps the speed of the filter catching up.
